packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-static-intervention-policy-group.vue

```diff
@@ -51,27 +69,58 @@
 				<section v-if="showNewValueOptions">
 					<h6 class="pt-4, pb-3">New Value</h6>
 					<div class="input-row">
-						<div v-for="(objective, index) in newValueInputs" :key="index" class="label-and-input">
-							<div class="label-and-input">
-								<label :for="objective">{{ NEW_VALUE_OPTIONS[index] }}</label>
-								<tera-input type="number" v-model="knobs[objective]" />
-							</div>
+						<div class="label-and-input">
+							<label>Lower bound</label>
+							<tera-input
+								type="number"
+								v-model="knobs.lowerBoundValue"
+								@update:model-value="$emit('update-self', knobs)"
+							/>
```

Comment:
- `tera-input` has a label parameter, you may want to check if it is more suitable here.
- Also you may want to use type=nist

Reply:
Good catch